geth/jail/execution_policy.go

```diff
@@ -32,26 +31,25 @@ func NewExecutionPolicy(
 }
 
 // Execute handles the execution of a RPC request and routes appropriately to either a local or remote ethereum node.
```

Comment:
`and routes appropriately to either a local or remote ethereum node` should be removed, this function doesn't do routing anymore.